src/App.tsx

```diff
@@ -1,61 +1,118 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { ChangeEventHandler, FormEventHandler, useState } from 'react';
+import { TodoList } from './components/TodoList';
+import { User } from './types/User';
+import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+
+const todosWithUsers = todosFromServer.map(todo => {
+  return {
+    ...todo,
+    user: usersFromServer.find(user => user.id === todo.userId) as User,
+  };
+});
 
 export const App = () => {
+  const [title, setTitle] = useState('');
+  const [userId, setUserDropdown] = useState<number>(0);
```

Comment:
it is better to name state consistently
like `[title, setTitle]` or `[userId, setUserId]`